pkg/metadata/host/host_tags.go

```diff
@@ -108,7 +108,7 @@ func GetHostTags(ctx context.Context, cached bool) *Tags {
 	providers := make(map[string]*providerDef)
 
 	if config.Datadog.GetBool("collect_ec2_tags") {
-		providers["ec2"] = &providerDef{1, ec2.GetTags, false}
+		providers["ec2"] = &providerDef{10, ec2.GetTags, false}
```

Comment:
Is there a reason to not have the same retry count for `ec2` and `gce`?
Does retrying 10 times could lead to issues. I am wondering why `retries` was not set to an higher value initially.

Reply:
I don't know the history here.  It looks like these were all 1, with a bump to 10 for enabled features, until d7dfbc25b16 when we only install providers for enabled features.  @vboulineau - maybe you can add more?

It's also worth noting that this parameter is really "tries" -- 1 means "call the API once", not "call the API, then retry once".

Retries are a [general recommended practice](https://docs.aws.amazon.com/general/latest/gr/api-retries.html) for AWS APIs.  That document doesn't specify MAX_RETRIES, but 10 is not at all unreasonable, and is unlikely to start interfering with customers' API limits.